### DISCUSSION TITLE
little confused with expired

### DISCUSSION THREAD

**Discussion by fearnie85** [gdbarron/VenafiTppPS#144] [General]
Hi,

I am just playing around/ testing a few things and I ran the following
`Find-TppCertificate -Path '\ved\policy' -Recursive -ExpireAfter (get-date).AddDays(-60)`

It returned circa 3000 expired certs.

However, when I check the webportal (dashboard - certificates - expired) it shows me 241.

Am I doing something wrong? Is there any way to filter out certificates which have been renewed?

I pulled one out at random and checked the certificate, and it shows as being renewed a few days ago.

In short, I am just trying to get a list of certificates which are on the expired list. Thanks.

Thanks in advance :)

**Comment by gdbarron** [comment]
Expire date is the date it expires.  If you want to see what has or will expire by a certain date, that's the date you use.  By getting certs that expire after a certain date, that includes all that expire from 60 days ago, not just from 60 days ago until now.  If you want to see what expired in the last 60 days, you need to use `-ExpireAfter` -60 and `-ExpireBefore` today or tomorrow.

**Reply by fearnie85** [reply]
oh I see. I get it now. sorry my bad :)

**Reply by fearnie85** [reply]
Sorry. I still see certificates which have renewed. Shown in the list when I pull these down. Is there anyway to filter out what has been done and what is still expired. 

Thanks.

**Reply by gdbarron** [reply]
Can you please post your code?
